.github/workflows/podvm_mkosi.yaml

```diff
@@ -242,6 +248,38 @@ jobs:
             make image
           fi
 
+      - name: Upload the qcow2 with oras
+        id: publish
+        working-directory: src/cloud-api-adaptor/podvm-mkosi
+        run: |
+          mkdir oras
+          cd oras
+          cp ../build/podvm-*.qcow2 .
+          tar cJf podvm.tar.xz podvm-*.qcow2
+          image=${{ inputs.registry }}/podvm-generic-fedora
+          if [ "${{inputs.debug}}" = "true" ]; then
+            image=${image}-debug
+          fi
+          tag=${{ steps.image_tag.outputs.image_tag }}
+          arch_tag=${tag}_${{ inputs.arch }}
+          oras push "${image}:${arch_tag}" podvm.tar.xz
```

Comment:
Suppose `${{ inputs.image_tag }}` isn't passed:

* builder: `${{ inputs.registry }}/podvm-builder-fedora-${{ inputs.arch }}:${{ github.sha }}`
* binaries: `${{ inputs.registry }}/podvm-binaries-fedora-${{ inputs.arch }}:${{ github.sha }}`
* podvm (oras) (not debug): `${{ inputs.registry }}/podvm-generic-fedora:${{ github.sha }}__${{ inputs.arch }}`
* podvm (oras) (debug): `${{ inputs.registry }}/podvm-generic-fedora-debug:${{ github.sha }}__${{ inputs.arch }}`

Is there a reason for podvm (oras) doesn't follow the builder/binaries pattern name? i.e, arch on the image's name instead of the tag.

If we name it as `${{ inputs.registry }}/podvm-generic-fedora-${{ inputs.arch }}:${{ github.sha }}` and push as oras then it may collide with the packer based image (which would have the same name, I think)?

Talking about packer builds... we publish, for example, `${{ inputs.registry }}/podvm-builder-fedora-${{ inputs.arch }}:${{ github.sha }}` and this mkosi workflow has a caching mechanism. It might end up using the built-on-packer-workflow builder image. Would it be a problem? Last time I check, IIRC, this isn't a problem but better ask than regret.

Reply:
I think you are right - the multi-arch approach isn't really working for decoupled builds, so I will go back to `podvm-fedora[-debug]-<arch>:tag`